calico-enterprise_versioned_docs/version-3.15/variables.js

```diff
@@ -15,7 +15,7 @@ const variables = {
   noderunning: 'calico-node',
   rootDirWindows: 'C:\\TigeraCalico',
   registry: 'quay.io/',
-  chart_version_name: 'v3.15.1-0',
+  chart_version_name: 'v3.16.2-0',
```

Comment:
Should this be v3.15.2?

Reply:
Not according to Dan Fox.

Reply:
That must be a misunderstanding, our helm chart versions always match the major.minor.patch  that you are releasing.

Reply:
For reference, you can always look here: https://github.com/tigera/calico-private/blob/release-calient-v3.15/calico/_data/versions.yml

Reply:
This was a typo on my end, sorry for the confusion! Yes, it should be 3.15.2 and not 3.16.2, sorry @bmckercher123!

Reply:
Fixed.